CI/integration_tests/NaCl/test_nacl_liquid.py

```diff
@@ -0,0 +1,203 @@
+"""
+Integration test with Liquid NaCl.
+
+Tests the following:
+* Project class function
+* Experiment class function
+* RDF calculator
+* Diffusion coefficients
+* Ionic Conductivity with and without charges.
+* Unwrapping with and without indices
+"""
+import mdsuite as mds
+import unittest
+import urllib.request
+import gzip
+import shutil
+import os
+import numpy as np
+import shutil
+
+
+class TestLiquidNaCl(unittest.TestCase):
+    """
+    Perform an integration test on liquid NaCl.
+    """
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        """
+        set up the project class and experiments.
+
+        Returns
+        -------
+        Updates class attributes.
+        """
+        cls.project = mds.Project(name='molten_nacl_test')
+        time_step = 0.002
+        temperature = 1400.0
+        base_url = 'https://github.com/zincware/ExampleData/raw/main/'
+        cls.endpoints = ['NaCl_gk_i_q.lammpstraj',
+                         'NaCl_gk_ni_nq.lammpstraj',
+                         'NaCl_i_q.lammpstraj',
+                         'NaCl_ni_nq.lammpstraj']
+        for item in cls.endpoints:
+            filename, headers = urllib.request.urlretrieve(f'{base_url}{item}.gz', filename=f'{item}.gz')
+            with gzip.open(filename, 'rb') as f_in:
+                with open(item, 'wb') as f_out:
+                    shutil.copyfileobj(f_in, f_out)
+            cls.project.add_experiment(experiment=item[:-11],
+                                       timestep=time_step,
+                                       temperature=temperature,
+                                       units='metal')
+            cls.project.experiments[item[:-11]].add_data(item)
+            cls.project.experiments[item[:-11]].set_charge("Na", 1)
+            cls.project.experiments[item[:-11]].set_charge("Cl", -1)
+            os.remove(item)
+            os.remove(f'{item}.gz')
+
+    @classmethod
+    def tearDownClass(cls) -> None:
+        """
+        Remove files generated.
+        Returns
+        -------
+        Removes the project directory.
+        """
+        shutil.rmtree('molten_nacl_test_MDSuite_Project', ignore_errors=True)
+
+    def test_completion(self):
+        """
+        Test that the project is loaded.
+        Returns
+        -------
+        assert that all the experiments exist.
+        """
+        reference = [item[:-11] for item in self.endpoints]
+        comparison = list(self.project.experiments.keys())
+        assert np.array_equal(reference, comparison)
```

Comment:
Is there a difference between this `assert np.array_equal` and `np.testing.assert_almost_equal` which you use later?